lib/mergeASRouterData.js

```diff
@@ -1,8 +1,640 @@
 const fs = require("fs");
 
-// NOTE: Add release versions here in descending order for all the JSON data
+// NOTE: Add release versions here for all the JSON data
 // that is available in `asrouter-local-prod-messages`
-const availableReleases = ["129", "128", "127", "126", "125", "124", "123"];
+let availableReleases = [130, 129, 128, 127, 126, 125, 124, 123];
+
+// Sort availableReleases in descending order
+// XXX This will need to be revisited if/once we add dot-versions
+availableReleases.sort(function (a, b) {
+  return b - a;
+});
+console.log(availableReleases);
+
+const manualMessages = [
```

Comment:
Can you clarify why this needs to be manual, and how we go about updating it with each new release?  My suspicion is that even if we need to do some kind of hack to get the data, that hack probably wants to go in the browser test itself, so that we always get the current version.

Reply:
Just talked to @AllegroFox and we figured out how to get the about welcome default message data into the browser test! I'll be pushing my changes soon

Reply:
I've rebased my new changes to the browser test and you can see them here https://phabricator.services.mozilla.com/D201646